Ginger/Ginger/ValueExpression/ValueExpressionEditorPage.xaml.cs

```diff
@@ -993,38 +988,77 @@ private void UpdateHelpForCSFunction(object sender, RoutedEventArgs e)
+        {
+            xHelpPanelText.Text = string.Empty;
+            TextBlockHelper TBH = new TextBlockHelper(xHelpPanelText);
+
+            TBH.AddUnderLineText("Help:");
+            TBH.AddLineBreak();
+
+            string msg = "Enter expression like " + GingerDicser.GetTermResValue(eTermResKey.Variable) + " from different levels, Environment Parameter or function which will be calculated in run time into dynamic value."
+                           + Environment.NewLine + Environment.NewLine +
+                           "Multiple expressions can be added together."; ;
+            TBH.AddText(msg);
         }
 
-        private void UpdateHelp(bool ShowHelpCategory, string Title, string Category, string HelpContentName, string HelpContent, string HelpExtraInfo = null)
+        private void ShowSpecificHelp(string title, string category, string helpContentName, string helpContent, string helpExtraInfo = null)
         {
-           
-            xWarningPanel.Visibility = Visibility.Collapsed;
-            xHelpPanel.Visibility = Visibility.Visible;
+            xHelpPanelText.Text = string.Empty;
+            TextBlockHelper TBH = new TextBlockHelper(xHelpPanelText);
 
-            if (ShowHelpCategory)
+            TBH.AddUnderLineText("Help:");
+            TBH.AddLineBreak();
+
+            TBH.AddBoldText(title);
+            if (!String.IsNullOrEmpty(category))
             {
-                xHelpCategoryPanel.Visibility = Visibility.Visible;
+                TBH.AddLineBreak();
+                TBH.AddText(category);
             }
-            else
+            if (!String.IsNullOrEmpty(helpContentName))
             {
-
-                xHelpCategoryPanel.Visibility = Visibility.Collapsed;
+                TBH.AddLineBreak();
+                TBH.AddText(helpContentName + ":");
             }
+            if (!String.IsNullOrEmpty(helpContent))
+            {
+                TBH.AddLineBreak();
+                TBH.AddText(helpContent);
+            }
+            if (!String.IsNullOrEmpty(helpExtraInfo))
+            {
+                TBH.AddLineBreak();
+                TBH.AddText(helpExtraInfo);
+            }
+        }
 
-            xHelpTitle.Content = Title;
-            XHelpCategory.Content = Category;
-            XHelpContentName.Text = HelpContentName + ": ";
-            XHelpContent.Text = HelpContent;
-            XHelpExtra.Text = HelpExtraInfo == null ? string.Empty : HelpExtraInfo;
+        private void ShowVBSWarnHelp()
+        {
+            xHelpPanelText.Text = string.Empty;
+            TextBlockHelper TBH = new TextBlockHelper(xHelpPanelText);
+
+            TBH.AddUnderLineText("Help:");
+            TBH.AddLineBreak();
+
+            string msg = "Value Expression Contains VBS Operation."
+                           + Environment.NewLine +
+                           "VBS operations are slow and works only on Windows OS, please consider changing it to CS expression."; ;
```

Comment:
Codacy found an issue: [Remove this empty statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6397283)